### ISSUE TITLE
Exercism Two-fer javascript problem with npm test

### ISSUE THREAD

**Issue by mohamedmehdigara** [codeHysteria28/Two-Fer#2]

**Comment by codeHysteria28** [comment]
Its related to your node modules in root level first run command npm install this will generate files you need to run the app

**Comment by mohamedmehdigara** [comment]
@GlobalGraphic 

1. npm install 👍 

2. npm test 👍  

3.  exercism submit two-fer.js 👎  (See error message in the screenshot)
![Capture](https://user-images.githubusercontent.com/47916299/101881436-b56dc280-3b94-11eb-827e-7bc87cd4b833.PNG)

**Comment by codeHysteria28** [comment]
Well you dont have a permissions to submit MY project so you better install the project from the excersim website dude

**Comment by mohamedmehdigara** [comment]
@GlobalGraphic It worked. But only after I copied your **node modules** folder and replacing your **two-fer.js** file with the one I downloaded from exercism.io. It seems I have a problem with **node modules** (specifically maybe with npm because npm install command has errors with my computer "Believe me I tried more than once and the node modules folder is still missing")  or the recent exercise folder from exercism.io has a problem with node modules.
Thanks for your kind help.
